ulog/README.md

```diff
@@ -1,16 +1,15 @@
 # Logging package
 
 `package ulog` provides an API wrapper around the logging library
-[zap](https://github.com/uber-go/zap). `package ulog` uses the builder pattern
-to instantiate the logger. With `LogBuilder` you can perform pre-initialization
+[zap](https://github.com/uber-go/zap). `package ulog` uses zap's builder
+to instantiate the logger. With zap's `Builder` you can perform pre-initialization
 setup by injecting configuration, custom logger, and log level prior to building
-the usable `ulog.Log` object.
+the usable `zapcore.Log` object.
 
-`ulog.Log` interface provides a few benefits:
+`ulog` provides a few benefits:
 
-- Decouple services from the logger used underneath the framework
-- Easy to use API for logging
-- Easily swappable backend logger without changing the service
+- Coupling with zap's Logger APIs.
+- Context based logging access via ulog.Logger(ctx)
```

Comment:
Suggest also pointing out that devs can use `ulog.Trace(ctx)` directly as a field.